dev/docker-compose.dev.full.yml

```diff
@@ -41,17 +41,18 @@ services:
     ports:
       - "53:53"
       - "443:443"
+      - "4510:4510"
```

Comment:
Would be good to have docs on which port is for what and linking to localstack docs